ios-xcframework/fastlane/Fastfile

```diff
@@ -77,3 +79,7 @@ def upload_to_a8c_s3(file:, name_on_s3:, prefix: 'gutenberg-mobile', skip_if_exi
     skip_if_exists: skip_if_exists
   )
 end
+
+def archive_name(id:)
+  "Gutenberg-#{id}.#{XCFRAMEWORK_ARCHIVE_TYPE}"
+end
```

Comment:
This function is called only in one place, what do you think moving it to its caller?